GetandTake/Services/Abstracts/ICategoryService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using GetandTake.Models;
+using GetandTake.Models.DTOs.BaseDTO;
+using GetandTake.Models.DTOs.ListDTO;
+
+namespace GetandTake.Services.Abstracts
+{
+    public interface ICategoryService
+    {
+        Task<IEnumerable<Category>> GetAllAsync();
```

Comment:
It is also a good practice in interfaces to add empty space between it's members